pkg/volume/csi/csi_block.go

```diff
@@ -103,7 +103,7 @@ var _ volume.CustomBlockVolumeMapper = &csiBlockMapper{}
 // Example: plugins/kubernetes.io/csi/volumeDevices/{specName}/dev
 func (m *csiBlockMapper) GetGlobalMapPath(spec *volume.Spec) (string, error) {
 	dir := getVolumeDevicePluginDir(m.specName, m.plugin.host)
-	klog.V(4).Infof(log("blockMapper.GetGlobalMapPath = %s", dir))
+	klog.V(4).InfoS(log("BlockMapper.GetGlobalMapPath"), "path", dir)
```

Comment:
What does the `log` function? Could you please remove it?

Reply:
The log() method currently adds "kubernetes.io/csi: " to the beginning of the log message. In my opinion, "kubernetes.io/csi" can be regarded as a label of the log, and there are too many callers, maybe we keep the status quo?

Reply:
We should remove usage of log function, we could consider adding a prefix to all log messages until we propose a better solution (logger naming)